cob_obstacle_distance/include/cob_obstacle_distance/obstacle_distance_data_types.hpp

```diff
@@ -49,4 +52,30 @@ struct ObstacleDistance
     uint32_t shape_type;
 };
 
+struct ST_Frame2CollisionMesh
+{
+    std::unordered_map<std::string, std::string> m;
+
+    ST_Frame2CollisionMesh()
+    {
+        m["arm_right_1_link"] = "package://schunk_description/meshes/lwa4p_extended/arm_1_collision.stl";
+        m["arm_left_1_link"] = "package://schunk_description/meshes/lwa4p_extended/arm_1_collision.stl";
+
+        m["arm_right_2_link"] = "package://schunk_description/meshes/lwa4p_extended/arm_2_collision.stl";
+        m["arm_left_2_link"] = "package://schunk_description/meshes/lwa4p_extended/arm_2_collision.stl";
+
+        m["arm_right_3_link"] = "package://schunk_description/meshes/lwa4p_extended/arm_3_collision.stl";
+        m["arm_left_3_link"] = "package://schunk_description/meshes/lwa4p_extended/arm_3_collision.stl";
+
+        m["arm_right_4_link"] = "package://schunk_description/meshes/lwa4p_extended/arm_4_collision.stl";
+        m["arm_left_4_link"] = "package://schunk_description/meshes/lwa4p_extended/arm_4_collision.stl";
+
+        m["arm_right_5_link"] = "package://schunk_description/meshes/lwa4p_extended/arm_5_collision.stl";
+        m["arm_left_5_link"] = "package://schunk_description/meshes/lwa4p_extended/arm_5_collision.stl";
+
+        m["arm_right_6_link"] = "package://schunk_description/meshes/lwa4p_extended/arm_6_collision.stl";
+        m["arm_left_6_link"] = "package://schunk_description/meshes/lwa4p_extended/arm_6_collision.stl";
+    }
+};
```

Comment:
Removed the mapping struct. Integrated URDF model and like suggested using the the Geometry data type.